File: EduChemSuite.API/EduChemSuite.API/Models/DistrictModel.cs
namespace EduChemSuite.API.Models;

public class DistrictModel : BaseModel
{
    public required String DistrictName { get; set; }
    public virtual ICollection<UserDistrictModel>? Administrators { get; set; }
    public virtual ICollection<DistrictSchoolsModel>? Schools { get; set; }
}

File: EduChemSuite.UI/UI/src/app/_models/DistrictModel.ts
import {BaseModel} from "./BaseModel";
import {UserDistrictModel} from "./UserDistrictModel";
import {DistrictSchoolsModel} from "./DistrictSchoolsModel";

export interface DistrictModel extends BaseModel{

  districtName?: string;
  administrators?: UserDistrictModel[];
  schools?: DistrictSchoolsModel[];
}

